packages/core/echo/echo-db/src/automerge/repo-proxy.ts

```
@@ -379,8 +424,8 @@ export class RepoProxy extends Resource {
       const isAbandoned = generation !== this._generation;
       if (!isAbandoned) {
         // Restore the state of pending updates if the RPC call failed.
-        addIds.forEach((id) => this._pendingAddIds.add(id));
-        removeIds.forEach((id) => this._pendingRemoveIds.add(id));
+        addIds.forEach((id) => this._pendingAddIds.add(id as DocumentId));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)